scripts/observability/import-openobserve-dashboard.mjs

```
@@ -0,0 +1,239 @@
+        options.password = argv[index + 1];
+        index += 1;
+        break;
+      case "--replace-by-title":
+        options.replaceByTitle = true;
+        break;
+      case "--dry-run":
+        options.dryRun = true;
+        break;
+      case "--help":
+        usage();
+        process.exit(0);
```

Biome (2.4.9)

[error] This case is falling through to the next case.

(lint/suspicious/noFallthroughSwitchClause)